core/deployment/src/main/java/io/quarkus/deployment/configuration/ConfigMappingUtils.java

```diff
@@ -121,8 +122,8 @@ private static void processConfigClass(
             generatedClassesNames.add(mappingMetadata.getClassName());
             // This is the generated implementation of the mapping by SmallRye Config.
             byte[] classBytes = mappingMetadata.getClassBytes();
-            generatedClasses.produce(new GeneratedClassBuildItem(isApplicationClass, mappingMetadata.getClassName(),
-                    classBytes));
+            generatedConfigClasses.put(mappingMetadata.getClassName(),
```

Comment:
Wonder if it's sufficient to check for `generatedConfigClasses.put() != null` instead of doing a `Stream.sorted().count()` in `AbstractJarBuilder`.

Reply:
I want the full list of problematic classes, otherwise it makes fixing issues very annoying.